lib/Analysis/AxisInfo.cpp

```diff
@@ -278,6 +278,9 @@ class AddSubOpAxisInfoVisitor final : public BinaryOpVisitorImpl<OpTy> {
 private:
   int64_t getContiguity(OpTy op, const AxisInfo &lhs, const AxisInfo &rhs,
                         int dim) override {
+    if (isa<arith::SubIOp>(op))
```

Comment:
It's a good catch, but I'd like to request two changes:
1. Inline comments that illustrate why we need this condition
2. Add a new test case here https://github.com/triton-lang/triton/blob/ca70f08835eee98747578cb0f37af2c033ecc60b/test/Analysis/test-alignment.mlir#L92

Reply:
Fixed.